### ISSUE TITLE
Timeshift cli segfaults during grub reinstall

### ISSUE THREAD

**Issue by michaldybczak** [teejee2008/timeshift#429]
When I want to restore from cli (tty or chroot) timeshift always crashes during grub reinstall step, no matter if I choose yes or not. I get info about "memory access violation" and "segfault", nothing more. The same restore action done from GUI (if possible) works correctly.

I had this problem a few times and luckily, one time I could restore via GUI, the other times I was able to fix the system so the system restore wasn't needed (but I didn't have GUI access). So at this point, not even once cli restore action was successful. Always the same, always in the same place.

**System:**
Operating System: Manjaro Linux 
KDE Plasma Version: 5.16.0
KDE Frameworks Version: 5.59.0
Qt Version: 5.12.4
Kernel Version: 5.1.11-1-MANJARO
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-6700HQ CPU @ 2.60GHz
Memory: 7,7 GiB

**Comment by scamdotnet** [comment]
Having the same issue.

**Comment by michaldybczak** [comment]
A lot of time has passed since the start of this case. I'll try to remember to do restore in cli when I will need it. GUI restores worked, so I have no recent data if this issue is still there or not for me.

@scamdotnet , what are your system stats and what version of timeshift are you using?

**Comment by scamdotnet** [comment]
Yes, this was in the CLI, and I ended up using the GUI and that worked. Since I restored on top of the partition that timeshift was on, I cannot get this data rn, but I do have to repeat the restore because I broke something with enabling grub.

**Comment by nadrog** [comment]
Having the same issue.
